crates/driver/rustfmt.toml

```diff
@@ -0,0 +1,19 @@
+tab_spaces = 4
+edition = "2021"
+fn_single_line = false
+format_code_in_doc_comments = true
+format_strings = true
+imports_layout = "HorizontalVertical"
```

Comment:
changes something that stable rustfmt already handles (but i'm fine with the change)